docs/actions.md

```diff
@@ -54,13 +54,14 @@ the [Docker](docker-actions.md) action or [native binary](docker-actions.md#crea
 paths more suitable. Multiple actions may be composed together to create a longer processing pipeline called a
 [sequence](#creating-action-sequences). A more advanced form of composition is described [here](conductors.md).
 
-* [JavaScript](actions-node.md)
-* [Python](actions-python.md)
+* [Go](actions-go.md)
```

Comment:
@akrabat @csantanapr I reordered these alphabetically. FYI.

Reply:
Alphabetical ordering makes sense to me.